### PR TITLE
consensus/istanbul: rename snapshot to roundState

### PR DESCRIPTION
**Pull request by tailingchen**

### CODE REVIEW

File: consensus/istanbul/backends/simple/api.go [hunk 1]
```diff
@@ -32,13 +32,13 @@ type API struct {
 
 // Snapshot returns current state and proposer
 func (api *API) Snapshot() {
```
Comment (bailantaotao) [inline consensus/istanbul/backends/simple/api.go]: name mismatch, RoundState?

Comment (markya0616) [inline consensus/istanbul/backends/simple/api.go]: rename RoundState()

File: consensus/istanbul/backends/simple/api.go [hunk 2]
```diff
@@ -32,13 +32,13 @@ type API struct {
...
-	state, snapshot := api.backend.core.Snapshot()
+	state, roundState := api.backend.core.RoundState()
 	p := api.backend.valSet.GetProposer().Address()
-	log.Info("Snapshot", "sequence", snapshot.Sequence, "Round", snapshot.Round,
+	log.Info("Snapshot", "sequence", roundState.Sequence, "Round", roundState.Round,
```
Comment (markya0616) [inline consensus/istanbul/backends/simple/api.go]: Snapshot -> RoundState

File: consensus/istanbul/backends/simple/api.go (hunk 1 above)
Comment (tailingchen) [inline consensus/istanbul/backends/simple/api.go]: fixed

Comment (tailingchen) [inline consensus/istanbul/backends/simple/api.go]: fixed

File: consensus/istanbul/backends/simple/api.go (hunk 2 above)
Comment (tailingchen) [inline consensus/istanbul/backends/simple/api.go]: fixed

File: consensus/istanbul/backends/simple/api.go [hunk 3]
```diff
@@ -30,15 +30,15 @@ type API struct {
 	backend *simpleBackend
 }
 
-// Snapshot returns current state and proposer
-func (api *API) Snapshot() {
-	state, snapshot := api.backend.core.Snapshot()
+// RoundState returns current state and proposer
+func (api *API) RoundState() {
```
Comment (markya0616) [inline consensus/istanbul/backends/simple/api.go]: Need to update `internal/web3ext/web3ext.go`

Comment (tailingchen) [inline consensus/istanbul/backends/simple/api.go]: fixed in `579227b`

**Pull request merged**